REDIS_PRODUCTION.md

```diff
@@ -0,0 +1,33 @@
+# Redis Memory Configuration for Production
+
+When deploying Redis in production environments, you should enable the memory overcommit setting to prevent potential issues with background saving and replication.
+
+## The Warning
+
+Redis may show this warning in logs:
+```
+WARNING Memory overcommit must be enabled! Without it, a background save or replication may fail under low memory condition.
+```
+
+## Linux Production Deployment
+
+For production deployments on Linux, you can:
+
+### Option 1: Use the production Docker Compose override file
+```bash
+docker-compose -f docker-compose.yml -f docker-compose.prod.yml up -d
+```
+
```

Comment:
This option won't work. Docker compose isn't able to set this value based on my testing.

If we still wanted an "option 1" we could mention the temporary commands that I listed in the discussion on this issue so we have an "option 1, temporary fix for current session" and "option 2: set up your system to permanently set this setting"